sami2py/utils.py

```diff
@@ -112,6 +115,37 @@ def set_archive_dir(path=None, store=True):
     else:
         raise ValueError('Path does not lead to a valid directory.')
 
+def return_fourier(x, coeffs):
+    """
+    Returns a Fourier series up to NumF coefficients
+    
+    Parameters
+    ----------
+    x : (1d ndarray)
+        solar local time in hours (slt)
+    coeffs : (array)
+        10x2 array of fourier coefficients
+        
+    Returns
+    --------
+    y : (array)
+        result of the fourier series 
+    """
+    def cos_a(x, n):
+        """simple cosine"""
+        return np.cos(n * np.pi * x / 12.0)
+        
+    def sin_a(x, n):
+        """simple sine"""
+        return np.sin(n * np.pi * x / 12.0)
+    
+    NumF = coeffs.shape
+    
+    y = coeffs[0]
+    for i in range(0, NumF[0]):
+        y += coeffs[i, 0]*cos_a(x, i) + coeffs[i, 1]*sin_a(x, i)
```

Comment:
Still running into a shape error here.  `y` should have the same number of points as `x`.
```suggestion
    y = 0.0 * x
    for i in range(0, NumF[0]):
        y += coeffs[i, 0]*cos_a(x, i) + coeffs[i, 1]*sin_a(x, i)
```